profiles/use.desc

```diff
@@ -101,7 +103,8 @@ ggi - Add support for media-libs/libggi (non-X video api/drivers)
 gif - Add GIF image support
 gimp - Build a plugin for the GIMP
 git - Enable git (version control system) support
-gles2-only - Use GLES 2.0 or later instead of full OpenGL
+gles2 - Enable GLES 2.0 support (independently of full OpenGL, see also: gles2-only)
```

Comment:
Maybe explain the acronym?

Reply:
Done. Not that it helps much, given how ambiguous these standards are in general.